src/api/subscriptions.js

```diff
@@ -640,3 +647,18 @@ export let SmartButtonOptions;
  * }}
  */
 export let SubscriptionRequest;
+
+/**
+ * @typedef {{
+ *  publisherProvidedId: string,
```

Comment:
do we also need an (optional?) initial set of product ID's and expire info here - otherwise we'll have to wait for the publisher to push an update before the link become effective.

Reply:
On reflection bad idea as it would introduce an abuse vector- we do need to document that the publisher must make the backend call to push subscription datas as soon as the link completes otherwise the use could be left wondering why the account they just linked can't access things.